tests/common/utils.py

```diff
@@ -305,6 +306,18 @@ def wait_for_condition(condition_checker, timeout=2, error_msg="Timeout", period
             raise TimeoutError(error_msg)
 
 
+async def wait_for_condition_async(
```

Comment:
I feel like we have many helpers like this...

Reply:
I made sync one, not sure async is present